src/lib/y2users/user_validator.rb

```diff
@@ -58,12 +58,6 @@ def issues
       err = Yast::UsersSimple.CheckFullname(user.full_name)
       add_fatal_issue(list, err, FULL_NAME_LOC)
 
-      return list unless user.password
-
-      user.password.issues.map do |issue|
-        list << issue
-      end
-
```

Comment:
@ancorgs,

I believe this is a dead code, isn't it? Notice the dot between `password` and `issues` which is making the validator crash when validating a user with a password. I was thinking if you simply wanted to write `password_issues`, but if I remember correctly we are performing such validation separated on purpose.

Please, check it.

Reply:
Ok, as discussed in a meeting, it was not dead but broken code. So, it has been restored and fixed. In addition, the ability to skip the password validations has been added through the 'skip_password` named param.